.unit-test/mocks/mocks.js

```diff
@@ -3,24 +3,92 @@
         GameBoardController: {
```

Comment:
Mocks should follow the usual name-casing convention of camelCase - it doesn't matter whether this matches the name of the dependency.